mobile/src/main/java/org/openhab/habdroid/background/ItemsControlsProviderService.kt

```diff
@@ -323,10 +352,12 @@ class ItemsControlsProviderService : ControlsProviderService() {
             else -> return null
         }
 
+        val location = getItemLocation(item, allItems).orEmpty()
+
         return if (stateful) {
             Control.StatefulBuilder(item.name, mainActivityPendingIntent)
                 .setTitle(item.label)
-                .setSubtitle(item.name)
+                .setSubtitle(location)
```

Comment:
Also set location in `setZone()`, so the long item list when adding items is divided into multiple pages?

Reply:
I'm torn here: on one hand grouping the long list makes sense, on the other hand, when implementing the suggestion to prefer equipment over location tags, it probably won't help much.

Reply:
Added another commit to my repo that combines both: use location for zone and subtitle for equipment.

Reply:
At least on my device (LineageOS 18) the item are grouped by zone only in the "Add elements" screen. After adding them, the zones are gone and therefore I don't see the location.

I pushed a commit that displays the location and equipment in the subtitle.

Reply:
> I pushed a commit that displays the location and equipment in the subtitle.

As commented, that doesn't match (at least) my style of tagging. As I doubt I'm the only one tagging that way, I wonder whether we could/should make this optional? Can one have settings activities for device controls?